src/containerapp/HISTORY.rst

```diff
@@ -5,14 +5,18 @@ Release History
 upcoming
 ++++++
 
+1.1.0b2
```

Comment:
Please don't bump version in this PR.
The Container App Extension release is managed by Container App team, which is more flexible. We usually release a new extension versions based on needs. Please contact Hong if you need to release a new features or fixes.
Before that, please follow the review process to review this PR.Thanks.